stdlib/public/SwiftShims/RefCount.h

```diff
@@ -92,24 +100,60 @@ class StrongRefCount {
 
   // Increment the reference count.
   void increment() {
+#if defined(COMPILER_HAS_ATOMIC_INTRINSICS)
     __atomic_fetch_add(&refCount, RC_ONE, __ATOMIC_RELAXED);
+#else
+    std::atomic<uint32_t> stdRefCount = ATOMIC_VAR_INIT(refCount);
```

Comment:
Wait, this doesn't make any sense. You need to increment the *real* refCount, not a local variable.

@gparker42, @jckarter, is there any problem with just making the real refcount field a std::atomic?